auth/jwt.go

```diff
@@ -50,4 +77,123 @@ type LocalJWTAuthProvider struct {
 	Keys       []jose.JSONWebKey `json:"keys"`
 }
 
+func (l *LocalJWTAuthProvider) verifyToken(ctx context.Context, token string, _ OIDCCallbackURLFunc) (*Identity, error) {
```

Comment:
This looks like it clones some functionality from coreos/go-oidc/verify.go.  What functionality is different for local JWT handling that would prevent us from creating an IDTokenVerifier and just using that code directly?
I'm concerned both about the potential for bugs introduced when cloning their logic, but also additional maintenance work to uptake fixes made in the library.